wp-content/themes/bpr2018/resources/templates/category.tpl.php

```diff
@@ -76,14 +76,16 @@ class="img-40 lazy-image"
                     </p>
       
                     <div class="post-author post-date font-size-20">
-                      <a href="<?php echo esc_url(get_author_posts_url(get_the_author_meta('ID'))); ?>">
-                        <?php the_author(); ?>
-                      </a>
+                      <span itemprop="author" hidden aria-hidden><?php the_author(); ?></span>
```

Comment:
Hm, since this is a span, can you check that it is still clickable and goes to the author's page?

Reply:
cool... I am figuring how to test this... but currently if I click the link on local host it leads me to a page just like the other stuff in the catergories